src/types.ts

```diff
@@ -6,7 +6,7 @@ import {
   EmitterWebhookEventName,
 } from "@octokit/webhooks";
 
-type ContextFactory<E extends EmitterWebhookEventName> =
+export type ContextFactory<E extends EmitterWebhookEventName> =
```

Comment:
```suggestion
type ContextFactory<E extends EmitterWebhookEventName> =
```

I don't think this is being used in any other files, right? Can probably be un-exported.